app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -205,12 +295,60 @@ class RepresenteeController @Inject() (
             if (answers.fold(_.id, c => Some(c.id)).contains(id))
               Redirect(routes.RepresenteeController.checkYourAnswers())
             else {
+              val name: Option[IndividualName] = answers.fold(i => i.name, c => Some(c.name))
```

Comment:
we don't want to proceed if there isn't a name. You could do something like this:
```scala
withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
  answers.fold(i => i.name, c => Some(c.name)) match {
     case None => Redirect(routes.RepresenteeController.checkYourAnswers()
     case Some(name) => ...
   }
}
```
You could do this for the display page endpoint as well and then pass in the name to the template where you need it.

Reply:
Done.